packages/server-wallet/src/handlers/update-channel.ts

```diff
@@ -39,16 +44,19 @@ const hasSupportedState = (cs: ChannelState): cs is ChannelStateWithSupported =>
 const ensureSupportedStateExists = (
   cs: ChannelState
 ): Either<UpdateChannelError, ChannelStateWithSupported> =>
-  hasSupportedState(cs) ? right(cs) : left(new UpdateChannelError(Errors.invalidLatestState));
+  hasSupportedState(cs)
+    ? right(cs)
+    : left(new UpdateChannelError(UpdateChannelError.reasons.invalidLatestState));
```

Comment:
:idea: Maybe on another iteration of the construction of errors, the `UpdateChannelError` class could have static constructors for error reason. Then, this could be refactored as:
```suggestion
    : left(UpdateChannelError.invalidLatestState());
```

Reply:
I thought about a similar idea, but then decided against it since, for every new error reason, we would have to add a static function. That being said, unit tests need to construct error instances. So, a static factory would be nice for that.